### PR TITLE
Change WordPress type to "app"

### PR DESCRIPTION
**Pull request by e1ee1e11**
Change WordPress type to "app"
Remove duplicate "odoo"
Update WordPress version
Update wordpress_icon URL

### CODE REVIEW

File: qpkgcenter_eng.xml [hunk 1]
```diff
@@ -466,6 +449,23 @@
 		</inspect>
 	</ctstation>
 	<ctstation>
+		<name>WordPress</name>
+		<internalName>wordpress</internalName>
+		<version>latest</version>
```
Comment (fcwu) [inline qpkgcenter_eng.xml]: It should have version

File: list.json [hunk 2]
```diff
@@ -303,6 +292,17 @@
         "type": "app"
     }, 
     {
+        "description": "WordPress is a rich content management system for creating blogs and websites with plugins, widgets, and themes.", 
+        "repository": "dockerhub", 
+        "arch": "amd64", 
+        "icon": "http://download.qnap.com/QPKG/images/QPKG/container/wordpress_icon.png", 
+        "displayName": "WordPress", 
+        "name": "wordpress", 
+        "version": "latest",
```
Comment (fcwu) [inline list.json]: Same here. Need version

**Comment by fcwu** [comment]
Please write exact version, not use latest

**Comment by e1ee1e11** [comment]
I update wordpress version to 4.4.2 with new docker-compose.yml file.
Please check [9831a0b](https://github.com/qnap-dev/container-apps/pull/20/commits/9831a0b4d230c2dabe38b488d6f6375d4d0174bc).
Thanks!

**Comment by e1ee1e11** [comment]
Update wordpress_icon URL

**Pull request merged**